mirrord/config/src/external_proxy.rs

```diff
@@ -27,6 +27,18 @@ pub static MIRRORD_EXTERNAL_TLS_KEY_ENV: &str = "MIRRORD_EXTERNAL_TLS_KEY";
 #[config(map_to = "ExternalProxyFileConfig", derive = "JsonSchema")]
 #[cfg_attr(test, config(derive = "PartialEq"))]
 pub struct ExternalProxyConfig {
+    /// ### external_proxy.listen {#external_proxy-listen}
```

Comment:
Think it would be helpful if you added a bit of reasoning on why you might need to set this up?